packages/app/cypress/e2e/sidebar_navigation.cy.ts

```diff
@@ -226,6 +247,34 @@ describe('Sidebar Navigation', () => {
       cy.get('[data-cy="app-header-bar"]').findByText('Settings').should('be.visible')
       cy.get('.router-link-active').findByText('Settings').should('be.visible')
     })
+
+    // TODO: Remove skip when we fix cy.reload() in Cypress in Cypress - UNIFY-1346
+    it.skip('resize nav and persist the state after refresh', () => {
+      cy.contains('fixture.js').click()
+
+      cy.get('.toggle-specs-text').click()
+
+      cy.intercept('mutation-Preferences_SetPreferences').as('setPreferences')
+
+      cy.get('[data-cy="reporter-panel"]').invoke('outerWidth').then(($initialWidth) => {
+        cy.get('[data-cy="panel2ResizeHandle"]').trigger('mousedown', { eventConstructor: 'MouseEvent' })
+        .trigger('mousemove', { clientX: 400 })
+        .trigger('mouseup', { eventConstructor: 'MouseEvent' })
+
+        cy.wait('@setPreferences')
```

Comment:
If not already covered elsewhere, it would be nice to have a test that validates the mutation separately from the reload so that we can get coverage of that.

Reply:
[fe55085](https://github.com/cypress-io/cypress/pull/20587/commits/fe55085dc8feec9e28a9db7cc57b66dba8cab4b3)